### PR TITLE
[Backtesting] update orders management and price channel

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/15
requires https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/17

### CODE REVIEW

File: octobot_trading/exchanges/exchange_simulator.py [hunk 1]
```diff
@@ -68,7 +69,10 @@ def __init__(self, config, exchange_type, exchange_manager, backtesting_data_fil
     async def create_backtesting_exchange_producers(self):
         for importer in self.exchange_importers:
             for updater in UNAUTHENTICATED_UPDATER_SIMULATOR_PRODUCERS:
-                await updater(get_trading_chan(updater.CHANNEL_NAME, self.exchange_manager.id), importer).run()
+                if updater == MarkPriceUpdater:
```
Comment (Herklos) [inline octobot_trading/exchanges/exchange_simulator.py]: Oh ok we discussed this change in #88 but it was from this PR... How should we proceed ? You want to merge this and modify this line in #88 or modify this PR that would be cleaner ?

Comment (GuillaumeDSM) [reply octobot_trading/exchanges/exchange_simulator.py]: I will remove this in this PR

File: octobot_trading/data/order.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 from asyncio import Lock
 
 import math
+
+from octobot_backtesting.api.backtesting import get_backtesting_current_time
```
Comment (Herklos) [inline octobot_trading/data/order.py]: I don't think it's a good idea to import backtesting module here.

File: octobot_trading/exchanges/exchange_simulator.py [hunk 3]
```diff
@@ -158,3 +158,6 @@ def get_trade_fee(self, symbol, order_type, quantity, price,
 
     def is_authenticated(self) -> bool:
         return False
+
+    def get_split_pair_from_exchange(self, pair) -> (str, str):
+        return split_symbol(pair)
```
Comment (Herklos) [inline octobot_trading/exchanges/exchange_simulator.py]: ?

File: octobot_trading/producers/simulator/recent_trade_updater_simulator.py [hunk 4]
```diff
@@ -17,12 +17,18 @@
...
 
 from octobot_commons.channels_name import OctoBotBacktestingChannelsName
+from octobot_commons.enums import PriceIndexes
+from octobot_trading.constants import OHLCV_CHANNEL
+from octobot_trading.enums import ExchangeConstantsOrderColumns
 from octobot_trading.producers.recent_trade_updater import RecentTradeUpdater
 
 
 class RecentTradeUpdaterSimulator(RecentTradeUpdater):
+    SIMULATED_RECENT_TRADE_LIMIT = 2
```
Comment (Herklos) [inline octobot_trading/producers/simulator/recent_trade_updater_simulator.py]: Can you explain why 2 recent trades to generate ?

Comment (GuillaumeDSM) [reply octobot_trading/producers/simulator/recent_trade_updater_simulator.py]: They are identical, it should change nothing to have 1 or 20 of them. 1 is not representative to me, at least 2 is better (can iterate) and I think more than 2 is unnecessary and takes time to process

File: octobot_trading/data/order.py (hunk 2 above)
Comment (GuillaumeDSM) [reply octobot_trading/data/order.py]: why?

Comment (GuillaumeDSM) [reply octobot_trading/data/order.py]: We can wrap it in exchange_simulator.

Comment (GuillaumeDSM) [reply octobot_trading/data/order.py]: Time managed in exchange and simulated exchange

File: octobot_trading/exchanges/abstract_exchange.pxd [hunk 5]
```diff
@@ -31,3 +31,5 @@ cdef class AbstractExchange(Initializable):
     cdef public ExchangeManager exchange_manager
 
     cdef public str name
+
+    cpdef float get_exchange_current_time(self)
```
Comment (Herklos) [inline octobot_trading/exchanges/abstract_exchange.pxd]: :+1:

**Pull request merged**